cfgov/jinja2/v1/_includes/organisms/complaint-maintenance-banner.html

```diff
@@ -0,0 +1,27 @@
+{# ==========================================================================
+
+   Complaint maintenance banner
+
+   ==========================================================================
+
+   Description:
+
+   Creates markup for a banner stating that the complaint intake form is 
+   offline for maintenance.
+
+   ========================================================================== #}
+
+{% import 'molecules/notification.html' as notification %}
+<div class="m-global-banner">
+  <div class="wrapper
+              wrapper__match-content">
+        {{ notification.render(
+            'warning',
+            true,
+            'We are currently undergoing planned maintenance.',
+            'Please come back '
+            + date_str + 
+            ' to submit or check the status of an existing complaint.'
```

Comment:
```suggestion
            ' to submit a complaint or check the status of an existing complaint.'
```

Reply:
Can we add this to the language that was already cleared?

Reply:
IMO, yes, this is a non-substantive grammatical correction.